moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -477,11 +495,15 @@ class Robot
 
   // Dynamic Reconfigure
   double speed_modifier_;
-
   moveit_simple_dynamic_reconfigure_Parameters params_;
-
   dynamic_reconfigure::Server
   <moveit_simple_dynamic_reconfigure_Config> dynamic_reconfig_server_;
+
+  // Kinematics
+  std::string ik_base_frame_;
+  std::string ik_tip_frame_;
+  bool ik_transforms_found_;
+  Eigen::Affine3d tool0_to_tcp_;
```

Comment:
Seems odd that you don't have a similar transform for the base?

Reply:
Done.